docs/development/Building in Windows.md

```diff
@@ -46,10 +46,13 @@ Continue with the Installation and accept all autodetected dependencies.
 
 ##Setup GNU ARM Toolchain
 
-----------
+Current (4.0.0 RC) Betaflight requires at least ```arm-none-eabi-gcc``` version ```7.3.1```, which can be obtained here:
 
-versions do matter, 5.4 is known to work well. Download this version from https://launchpad.net/gcc-arm-embedded/5.0/5-2016-q2-update/+download/gcc-arm-none-eabi-5_4-2016q2-20160622-win32.zip 
+ - https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads (7-2018-q2-update)
 
+Or via direct link:
+
+ - https://armkeil.blob.core.windows.net/developer/Files/downloads/gnu-rm/7-2018q2/gcc-arm-none-eabi-7-2018-q2-update-win32.zip
```

Comment:
Who is this link hosted by? This does not look like it is something that is officially released by ARM.

Also, if we are linking to the Windows version of the suite, we should also provide links for the MacOS / linux versions.

Reply:
That link comes directly from here:

https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads

Reply:
Additionally, this is edited in the `building in Windows`  file, hence just the windows ZIP link :)

Reply:
Fair enough.